src/NIfTI.jl

```diff
@@ -406,9 +406,16 @@ end
 
 # Look for a gzip header in an IOStream
 function isgz(io::IO)
-    ret = read(io, UInt8) == 0x1F && read(io, UInt8) == 0x8B
-    seek(io, 0)
-    ret
+    try
+        ret = read(io, UInt8) == 0x1F && read(io, UInt8) == 0x8B
+        seekstart(io)
```

Comment:
Also minute change, changed `seek(io, 0)` to `seekstart(io)`

Reply:
Nice catch